dpnp/dparray.pyx

```diff
@@ -942,6 +942,34 @@ cdef class dparray:
         """
         return argsort(self, axis, kind, order)
 
+    def partition(self, kth, axis=-1, kind='introselect', order=None):
```

Comment:
It is not quite clear for me, why we need this code here. Why we can't call dpnp.partition() instead implementation it here in dparray.